appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -407,7 +408,8 @@
            ;; for event handlers and component names.
            (let ((registeredObject (string->symbol registeredComponentName)))
                  (if (is-bound-in-form-environment registeredObject)
-                     (if (eq? (lookup-in-form-environment registeredObject) componentObject)
+                     ;; temporarily deleted because I have no ideas how to get true out of this condition using isMulti
+                     ;;(if (or (eq? (lookup-in-form-environment registeredObject) componentObject) (isMulti))
```

Comment:
Yea, I knew it was a workaround, but I only attempted this way because I was blocked on other things in runtime.scm. I thought I'd see what you all thought about this, but I guess I will try to go back to the way that I was trying to do things.